fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -394,7 +394,6 @@ public BrokerFileSystem getS3AFileSystem(String path, Map<String, String> proper
         String disableCache = properties.getOrDefault(FS_S3A_IMPL_DISABLE_CACHE, "true");
         String s3aUgi = accessKey + "," + secretKey;
         FileSystemIdentity fileSystemIdentity = new FileSystemIdentity(host, s3aUgi);
-        cachedFileSystem.putIfAbsent(fileSystemIdentity, new BrokerFileSystem(fileSystemIdentity));
```

Comment:
Why remove this?

Reply:
@morningman Because this step has been done in the updateCachedFileSystem method in the next line of code, this line of code is redundant.